charts/timber-observation-service/values.yaml

```diff
@@ -173,7 +173,7 @@ fluentd:
   # -- Flag to toggle deployment of Observation Service fluentd
   enabled: false
 
-  nameOverride: timber-observation-service-fluentd
```

Comment:
Would `timber-obs-fluentd` be a better name, considering there could be multiple fluentd being deployed in the same namespace?

Reply:
yea fluentd might clash since the chart is name to fluentd, will change to `obs-fluentd` 😅